src/screens/game_map.py

```diff
@@ -345,7 +345,7 @@ def _setup_collideable_tile(
             pf_add_matrix_collision(
                 self._pf_matrix,
                 (pos[0] / SCALE_FACTOR, pos[1] / SCALE_FACTOR),
-                surf.size,
+                surf.get_size(),
```

Comment:
- This change is not necessary. `pygame.Surface` defines `size` as a read-only property, i.e. effectively as a shortcut for this